docs/release-notes/8.3.asciidoc

```diff
@@ -55,22 +75,26 @@ IMPORTANT: To restore custom and prebuilt rules, you need privileges to <<enable
 . Select the same rules, then click *Bulk actions -> Enable*.
 +
 NOTE: After you've re-enabled the affected rules, the rules' *Last Response* values will change to `Pending` and then update to `Active` or `OK`.
-
++
 . Go to the next page of results in the rules table and repeat steps 1 through 6.
 
 *Restore affected custom rules only (optional)*
 
-NOTE: This is an alternative option for users who have only enabled custom rules and or duplicated and enabled pre-built rules.
+NOTE: This is an alternative option for users who have only enabled custom rules and or duplicated and enabled prebuilt rules.
 
 . Go to the *Rules* page (*Detect -> Rules*) and click *Elastic rules*.
 . Switch on the *Technical preview* toggle above the table.
 . In the rules table, click *Custom rules*.
 . Sort the rules table by the *Last Response* column to show the latest rule statuses.
 . Select rules with the `Failed` status, then click *Bulk actions -> Tags -> Add Tags*.
 . Add a new tag, for example `rules_to_fix`. This will generate new API keys and resolve the bug.
-
++
 On the next scheduled rule execution, the *Last Response* value for the rule will change to `Pending`, and then to `Active` or `OK`.
 
+*The `matches` operator is not supported for rule exceptions*
+
+* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting the unsupported exception and refreshing the rule ({issue}136340[#136340]).
```

Comment:
```suggestion
* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting the unsupported exceptions and refreshing the rules ({issue}136340[#136340]).
```